### ISSUE TITLE
wxPython interface Issues on OSX

### ISSUE THREAD

**Issue by jorgerobles** [bq/horus#7]
There are a bunch of GUI issues on OSX, running wxPython 3.0.2.0-cocoa. 
This is a placeholder.

**Comment by Jesus89** [comment]
I use in GNU/Linux Ubuntu wxPython 2.8. Please, try this version on OSX.

**Comment by jorgerobles** [comment]
The wx 2.8 is no more on OSX. The minimum version is wxPython 2.9.5.0-cocoa
It throws this error (among other blocked GUI)
```
 File "/usr/local/lib/wxPython-2.9.5.0/lib/python2.7/site-packages/wx-2.9.5-osx_cocoa/wx/_misc.py", line 1324, in Start
    return _misc_.Timer_Start(*args, **kwargs)
PyAssertionError: C++ assertion "wxThread::IsMain()" failed at /BUILD/wxPython-src-2.9.5.0/src/common/timerimpl.cpp(61) in Start(): timer can only be started from the main thread
```

**Comment by Jesus89** [comment]
Ok. We had the same issue in windows. Just include Mac in the condition to use timer without Thread: https://github.com/bq/horus/blob/develop/src/horus/gui/util/imageView.py#L133

**Comment by jorgerobles** [comment]
Almost there!

**Comment by jorgerobles** [comment]
I'm stuck at one big interface problem. All modal close events are not dispatched to parent windows.
The modals are closed, but only visually.

**Comment by Jesus89** [comment]
Solved here: 3d233a852d351fd4c8f4c3c8b037f680027112e8

**Comment by honda10** [comment]
I cannot calibrate the lasers during wizard mode. they will work at the end of set up.
I am using windows 8.1 with a baud BT.ATmega328
any help please
Honda
